Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
@marcopompei fascinating! my first impression is that the custom one is more intelligible. looking at those values though - i think it would be `higherThan: AdditivePrecedence`, to keep it the same. is it preferable to choose `CastingPrecedence` do you think?